source/_components/rainbird.markdown

```diff
@@ -29,6 +29,8 @@ To enable it, add the following to your `configuration.yaml` file:
 rainbird:
   host: IP_ADDRESS_OF_MODULE
   password: YOUR_PASSWORD
+  trigger_time: 6
```

Comment:
`trigger_time` is optional as well right?

Reply:
It should mandatory. To set irrigation time.

It could be then overridden for specific zones, but has to be set for some default.

Reply:
It isn't mandatory according to the code: <https://github.com/home-assistant/home-assistant/pull/26393/files#diff-d575fd59bad63f44b30cd5e8ef6286baR44>

Reply:
Now it is